src/NLog/LayoutRenderers/LevelLayoutRenderer.cs

```diff
@@ -47,21 +48,40 @@ namespace NLog.LayoutRenderers
     [ThreadSafe]
     public class LevelLayoutRenderer : LayoutRenderer, IRawValue, IStringValueRenderer
     {
+        private static readonly Dictionary<int, string> _upperCaseMapper = new Dictionary<int, string>()
+        {
+            { LogLevel.Trace.Ordinal, LogLevel.Trace.ToString().ToUpperInvariant() },
+            { LogLevel.Debug.Ordinal, LogLevel.Debug.ToString().ToUpperInvariant() },
+            { LogLevel.Info.Ordinal, LogLevel.Info.ToString().ToUpperInvariant() },
+            { LogLevel.Warn.Ordinal, LogLevel.Warn.ToString().ToUpperInvariant() },
+            { LogLevel.Error.Ordinal, LogLevel.Error.ToString().ToUpperInvariant() },
+            { LogLevel.Fatal.Ordinal, LogLevel.Fatal.ToString().ToUpperInvariant() },
+            { LogLevel.Off.Ordinal, LogLevel.Off.ToString().ToUpperInvariant() },
+        };
+
         /// <summary>
         /// Gets or sets a value indicating the output format of the level.
         /// </summary>
         /// <docgen category='Rendering Options' order='10' />
         [DefaultValue(LevelFormat.Name)]
         public LevelFormat Format { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether upper case conversion should be applied.
+        /// </summary>
+        /// <value>A value of <c>true</c> if upper case conversion should be applied otherwise, <c>false</c>.</value>
+        /// <docgen category='Rendering Options' order='10' />
+        [DefaultValue(true)]
```

Comment:
Is the default really true?

Reply:
test is green, so I mis something?

Reply:
It should be false. So it is strange that the test is green. Will take a look

Reply:
The test is not green. The test is located in AppVeyor build. Guess the `VerifyDefaultValues`-test should be extracted from SourceCodeTests.cs and be placed as normal unit-test.